front/app/resources/GetUsers.tsx

```diff
@@ -24,16 +24,24 @@ export type Sort =
   | '-role';
 
 export interface InputProps {
+  // Not used
```

Comment:
Do we want to deprecated them?

Reply:
Just leaving the note for the person who works on converting this to React Query.